fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -1177,4 +1177,25 @@ public void testAppendOnlyToBinary() throws IOException {
         assertEquals(HttpStatus.SC_FORBIDDEN, getStatus(postReq));
     }
 
+    @Test
+    public void testFoafAgent() throws IOException {
+        final String path = ingestObj("/rest/foaf-agent");
+        ingestAcl("fedoraAdmin", "/acls/03/foaf-agent.ttl", path + "/fcr:acl");
+        final String username = "user1";
+
+        final HttpGet req = new HttpGet(path);
+        setAuth(req, username);
+        assertEquals(HttpStatus.SC_OK, getStatus(req));
+    }
+
+    @Test
+    public void testAuthenticatedAgent() throws IOException {
+        final String path = ingestObj("/rest/authenticated-agent");
+        ingestAcl("fedoraAdmin", "/acls/03/authenticated-agent.ttl", path + "/fcr:acl");
```

Comment:
For consistency, I would suggesting putting these two new TTL files in `/acls/25/`.